frontend/src/types.ts

```diff
@@ -2979,6 +2979,7 @@ export enum NotebookNodeType {
     FeatureFlag = 'ph-feature-flag',
     Person = 'ph-person',
     Link = 'ph-link',
+    Backlink = 'phBacklink',
```

Comment:
can we make this ph-back-link? No strong reason why, just keeps consistency with the others

Reply:
For sure, will change it back. I vaguely remember Tiptap breaking internally somewhere because of the dashed-case. Probably something I ended up not using, but might have to loop back and update all of these if that rears its head again